src/components/Link/Link.tsx

```diff
@@ -25,7 +25,7 @@ export interface LinkProps extends DOMProps, QAProps {
 
 const b = block('link');
 
-const PureLink = React.forwardRef<HTMLAnchorElement | HTMLSpanElement, LinkProps>(function Link(
+export const Link = React.forwardRef<HTMLElement, LinkProps>(function Link(
```

Comment:
This can break types

Reply:
`withEventBrokerDomHandlers.tsx` uses `React.forwardRef<HTMLElement, ...>` to wrap components.
So not changing types to `React.forwardRef<HTMLElement, ...>` breaks types.

Reply:
Oh, ok